x-pack/plugins/cloud_security_posture/public/pages/benchmarks/use_csp_benchmark_integrations.ts

```diff
@@ -41,9 +41,22 @@ export const useCspBenchmarkIntegrations = ({
   return useQuery(
     [QUERY_KEY, query],
     () =>
-      http.get<ListResult<Benchmark>>(BENCHMARKS_ROUTE_PATH, {
+      http.get<GetBenchmarkResponseV1>(BENCHMARKS_ROUTE_PATH, {
         query,
-        version: BENCHMARKS_API_CURRENT_VERSION,
+        version: '1',
+      }),
+    { keepPreviousData: true }
+  );
+};
+
+export const useCspBenchmarkIntegrationsV2 = () => {
+  const { http } = useKibana().services;
+
+  return useQuery(
+    [QUERY_KEY],
```

Comment:
Pay attention you use the same query key here, have you verified it doesn't conflict?

Reply:
I havent seen it fail so far. the V1 version of this is only being used on 1 instance(when clicking Add Agent button from NotDeployed component) and I didn't seem to see any errors on that instance. V1 and V2 are never used at the same time so I don't think this will cause issue ? But to be safe we can just add another Query key (if there's no down side of doing so of course)